src/charm.py

```diff
@@ -378,7 +378,8 @@ def _on_peer_relation_changed(self, event: RelationChangedEvent) -> None:
             self.join_unit_to_cluster()
             for port in ["3306", "33060"]:
                 try:
-                    subprocess.check_call(["open-port", f"{port}/tcp"])
+                    # TODO use set_ports instead
+                    subprocess.check_call(["open-port", f"{port}/tcp"])  # noqa: S603 S607
```

Comment:
Ops `set_ports` should be able to do the same thing without having to call external utilities.